guides/learning_guide.rst

```diff
@@ -0,0 +1,1099 @@
+CPU cores—now also with 6 GPU resources (-g6). When this request is granted,
+Flux automatically creates a child instance that manages the requested resources.
+This job script then uses this instance as, in effect, a dedicated supercomputer and
+submits a series of sub-batch scripts to it. At line 2, the command run.dock.sh is
+again submitted, requesting a subset of the resources: 254 compute nodes with 24
+CPU cores each, and no GPU resources. When this script is assigned to this subset,
+it will again spin up an even smaller Flux instance and run many smaller docking
+simulation applications under it. Similarly, at line 3, another sub-job script is submitted
+to this instance with a request of 192 nodes each with 18 CPU-core and all of the
+GPU resources—this time to execute a combined set of molecular dynamics (MD)
+simulations along with data analytics applications. Line 4 initiates the third sub-job
+script that runs AI applications to accelerate certain part of this computation with
+a request of 64 nodes, again each with 18 CPU-core and all of the requested GPU
+resources. At line 5, flux queue directs the top-level batch script to wait until
+all three sub-jobs will complete. To accomplish this under other more traditional
+products, each different workflow team either develops its own ad hoc software or
+relies ona  separate workflow management software. The efficiency, robustness,
+and scalability of these software solutions vary wildly; the user of each workflow
+would need to find and read the documentation, learn about new interfaces, then
+install and deploy them. These examples can become even more complex. What if
+the first job needed to connect to, and remotely submit tasks to, the second job?
+However, with Flux, such communication and coordination between distinct batch
+jobs can be easily accomplished using the same interfaces—in fact, the user needs
+only make a single line change to the first script in order to submit a new task such
+as an AI re-training task remotely into the Flux instance managing the second job:
+e.g., inserting ``FLUX)URI=$(JobID_3b_AI.sh) flux batch retrain.sh`` into
+a line between 2 and 3 of Figure 3a. Based on this request, the first job uses the
+connector built into the Flux instance of the second job, then channels the same Flux
+commands to batch additional tasks or retrieve the status of the simulation run.
+
+(a) Conventional batch job script (Figure 3a)
+
+.. :: code-block console
+
+    flux batch -N256 -n256 -c42 --wrap <<SCRIPT
+      flux run -N256 -n10752 sim.app
+    SCRIPT 
+
+(b) Emerging batch job script  (Figure 3b)
+
+
+.. :: code-block console
+
+    flux batch -N256 -n256 -c42 -g6 --wrap <<SCRIPT
+      flux batch -N256 -c24 run.dock.sh
+      flux batch -N192 -c18 -g6 run.MD.sh
+      flux batch -N64 -c12 -g6 run.AI.sh
+      flux queue drain
+    SCRIPT 
+
+In the above, compared to a conventional request (a) a job request for an emerging HPC workflow (b)
```

Comment:
This paragraph doesn't display as  "Figure 3" sub-image kinda thing.  Not sure how to do that in rst off the top of my head.